addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -978,6 +978,19 @@ def test_message_update_content_check(self):
                     message, "<p>New Body</p>"
                 )
 
+    @users("employee")
+    def test_message_get_suggested_recipients_multiemail(self):
```

Comment:
:abc: 

Also if it exists in this version can you add a case in `test_track_suggested_recipients` instead?